extensions/amp-position-observer/0.1/test-e2e/test-target-id.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'amp-position-observer target in AMPHTML ad',
+  {
+    testUrl:
+      'http://localhost:8000/test/fixtures/e2e/amp-position-observer/target-id.html',
+    environments: 'amp4ads-preset',
```

Comment:
are you able to enable this test for all environments? try ampdoc-amp4ads-preset

Reply:
This test assumes a creative rendered in a specific position inside a 300x250 envelope, and the envelope itself is being scrolled. It would not work on other shaped envelopes.